src/main/kotlin/fi/metatavu/metaform/server/controllers/AuditLogEntryController.kt

```diff
@@ -18,6 +19,10 @@ class AuditLogEntryController {
     @Inject
     lateinit var auditLogEntryDAO: AuditLogEntryDAO
 
+    @Inject
+    @ConfigProperty(name = "metaforms.features.auditlog", defaultValue = "true")
+    var auditLog: Boolean = true
```

Comment:
```suggestion
    lateinit var auditLog: Boolean
```
Quarkus will inject the defaultValue from `@ConfigProperty` so initializing the variable shouldn't be needed.